hr_timesheet_menu_filters/views/hr_timesheet_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_timesheet.act_hr_timesheet_line" model="ir.actions.act_window">
+        <field name="search_view_id" ref="hr_timesheet.hr_timesheet_line_search"></field>
+        <field name="context">{'search_default_this_week':1}</field>
```

Comment:
Please look at original Odoo code:

https://github.com/odoo/odoo/blob/15.0/addons/hr_timesheet/views/hr_timesheet_views.xml#L258

- `search_view_id` is already filled, IMO this line should be removed.
- `hr_timesheet` module already has a `search_default_week` context value. How does it work?